src/display/api.js

```diff
@@ -2476,10 +2487,15 @@ class WorkerTransport {
       renderingIntent += RenderingIntentFlag.OPLIST;
     }
 
+    const { ids: modifiedIds, hash: modifiedIdsHash } =
+      annotationStorage.modifiedIds;
+
     return {
       renderingIntent,
-      cacheKey: `${renderingIntent}_${annotationStorageSerializable.hash}`,
+      cacheKey: `${renderingIntent}_${annotationStorageSerializable.hash}_${isEditing}_${modifiedIdsHash}`,
```

Comment:
How about keeping the `isEditing`-part a little shorter and with a constant length?
```suggestion
      cacheKey: `${renderingIntent}_${annotationStorageSerializable.hash}_${isEditing ? 1 : 0}_${modifiedIdsHash}`,
```

Reply:
Also, given the amount of things included in the cacheKey this is perhaps less readable than before?
Do we want to add something like this (just above):
```javascript
const cacheKeyBuf = [
  renderingIntent,
  annotationStorageSerializable.hash,
  isEditing ? 1 : 0,
  modifiedIdsHash,
];
```
since we could then change this line to:
```suggestion
      cacheKey: cacheKeyBuf.join("_"),
```